tests/conftest.py

```diff
@@ -88,18 +89,6 @@ def url_safe_serializer(secret_key) -> URLSafeSerializer:
     return URLSafeSerializer(secret_key)
 
 
-@pytest.fixture()
```

Comment:
Maybe that one also could be better in a separate PR (for trackability)?

Reply:
Yes, we could